src/main/java/me/dinowernli/grpc/polyglot/config/CommandLineArgs.java

```diff
@@ -16,107 +16,95 @@
 
 import com.google.common.collect.ImmutableMultimap;
 import com.google.common.collect.Maps;
-import org.kohsuke.args4j.CmdLineException;
-import org.kohsuke.args4j.CmdLineParser;
-import org.kohsuke.args4j.Option;
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.ParameterException;
+import com.beust.jcommander.Parameters;
 
 /** Provides easy access to the arguments passed on the command line. */
+@Parameters(separators = "=")
```

Comment:
Would it make sense to support spaces here as well?

Reply:
Sure it does. Done.